README.md

```diff
@@ -1,3 +1,6 @@
+[![Template Check Status]https://github.com/carpentries/lesson-example/workflows/template/badge.svg)](https://github.com/carpentries/lesson-example/actions/workflows/template.yml?query=workflow%3A%22Test+template%22)
```

Comment:
I would remove this since it is specific to the styles repo.

```suggestion
```

Reply:
Done in b9d1732. Rebased on the tip of `gh-pages` then push-forced (rebasing interactively to amend the previous commit message turned out to be hairy).

Reply:
Sorry about that. I updated this branch with the `gh-pages` without asking your permission first to resolve conflicts.

Reply:
No worries; I could have added another commit to apply the suggestion, but I tend to avoid changes that are undone in a subsequent commit if possible to keep a cleaner commit history.